### PR TITLE
Update x12-edi-connector.adoc

### PR DESCRIPTION
**Pull request by mcherem**
Added file size support note.

### CODE REVIEW

**Review by amitfernandes — APPROVED** [review]
Added language related to connector and file size limitation.

File: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
-_Premium_
+Support Category: Premium
+X12 EDI Connector Version 2.1.1
 
-The X12 EDI Connector lets you convert X12 messages to and from
-DataWeave-compatible representations using lists and maps. X12 versions 003010, 003020, 003030, 003040, 003050, 003060, 003070, 004010, 004020, 004030, 004040, 004050, 004060, 005010, 005020, 005030, 005040, 005050, and 006020 are supported with supplied schema definitions for all standard transaction sets.
-Many HIPAA document versions are also supported, as listed separately in the
-Connector HIPAA reference page.
+The Anypoint X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.
```
Comment (valkyrie69) [inline modules/ROOT/pages/x12-edi/x12-edi-connector.adoc:10]: Per Kim, naming convention for connectors is:
Anypoint Connector for X12 EDI
Then short version X12 EDI Connector

File: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc [hunk 2]
```diff
@@ -21,6 +22,9 @@ release notes is listed in the See Also section of this document.
 To use the X12 EDI connector in a production environment, you must
 have purchased a MuleSoft license for Anypoint B2B.
 
+NOTE: This connector effectively supports files up to 20MB. As you process files beyond 20MB, consider that the connector has a 40x memory requirement corresponding to the EDI file size.
```
Comment (valkyrie69) [inline modules/ROOT/pages/x12-edi/x12-edi-connector.adoc]:
```suggestion
NOTE: X12 EDI Connector supports files up to 20MB. If you process files larger than 20MB, consider that the connector has a 40x memory requirement that corresponds to the EDI file size. 
```

File: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc [hunk 3]
```diff
@@ -21,6 +22,9 @@ release notes is listed in the See Also section of this document.
 To use the X12 EDI connector in a production environment, you must
 have purchased a MuleSoft license for Anypoint B2B.
 
+NOTE: This connector effectively supports files up to 20MB. As you process files beyond 20MB, consider that the connector has a 40x memory requirement corresponding to the EDI file size. 
+For example, a 30MB X12 (or EDIFACT) file needs 1.4GB of memory. It's important to consider this memory requirement in conjunction with your TPS needs for large files.
```
Comment (valkyrie69) [inline modules/ROOT/pages/x12-edi/x12-edi-connector.adoc]:
```suggestion
For example, a 30MB X12 (or `EDIFACT`) file needs 1.4GB of memory, so it's important to consider this memory requirement in conjunction with your TPS needs for large files.
```

**Review by valkyrie69 — APPROVED** [review]
Minor feedback.

File: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/x12-edi/x12-edi-connector.adoc:10]: I see you have changed this in the first sentence - should this also be changed in the title up top?

Comment (mcherem) [reply modules/ROOT/pages/x12-edi/x12-edi-connector.adoc:10]: Not in titles, got it.

**Pull request merged**